src/PriorityMap.php

```diff
@@ -54,7 +54,12 @@ public function set($key, $value, $priority = 0)
     public function get($key)
     {
         $key = $this->getScalarKey($key);
-        return isset($this->list[$key]) ? $this->list[$key]->value : null;
+
+        if(empty($this->list[$key])) {
```

Comment:
PSR-2, why do you change isset to empty?

Reply:
Ok, I use this because I misunderstand the ```get``` method.
Now I add this previous code back.